tf_encrypted/convert/register.py

```diff
@@ -125,6 +136,12 @@ def sigmoid(converter: Converter, node: Any, inputs: List[str]) -> Any:
 
 def strided_slice(converter: Converter, node: Any, inputs: List[str]) -> Any:
     input = converter.outputs[inputs[0]]
+
+    if isinstance(input, tf.NodeDef):
```

Comment:
what can `input` be here if not `NodeDef`?

Reply:
if it's not a NodeDef it would be already a private pond tensor.